### ISSUE TITLE
REQUEST: Netlify site for kubernetes-sigs/cluster-api-operator

### ISSUE THREAD

**Issue by furkatgofurov7** [kubernetes/org#4781]
### Repository

kubernetes-sigs/cluster-api-operator

### Project domain

cluster-api-operator.sigs.k8s.io

### Additional context for request

The PR that adds netlify.toml configuration file: https://github.com/kubernetes-sigs/cluster-api-operator/pull/433
Tracking issue: https://github.com/kubernetes-sigs/cluster-api-operator/issues/434

**Comment by furkatgofurov7** [comment]
@mrbobbytables @cblecker hey folks, can you please help with setting up Netlify site?

/assign @cblecker @mrbobbytables

**Comment by furkatgofurov7** [comment]
Also opened https://github.com/kubernetes/k8s.io/pull/6462 for DNS record

**Comment by cblecker** [comment]
/unassign @mrbobbytables @cblecker 

We need the configuration merged into the target repo first before we can do any configuration. Thanks!

**Comment by cblecker** [comment]
This is now complete. The site should be available at https://cluster-api-operator.sigs.k8s.io/ with a signed TLS certificate provisioning automatically within the next 24 hours.

/close
